src/features/content-insights/charts/bar/Bar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+
+import { Direction } from '../types';
+
+import './Bar.scss';
+
+const DEFAULT_SIZE = 50;
+
+interface Props {
+    color?: string;
+    direction?: Direction;
+    onMouseEnter?: () => void;
+    onMouseLeave?: () => void;
+    size: number;
+}
+
+function Bar({ color, direction = 'vertical', onMouseEnter = noop, onMouseLeave = noop, size = DEFAULT_SIZE }: Props) {
+    const isHorizontal = direction === 'horizontal';
+    const cssProperty = direction === 'horizontal' ? 'width' : 'height';
+    const [style, setStyle] = React.useState({
+        backgroundColor: color,
+        [cssProperty]: '0%',
+    });
+
+    const adjustedSize = Math.max(0, size);
+
+    React.useEffect(() => {
+        setStyle({
+            backgroundColor: color,
+            [cssProperty]: adjustedSize === 0 ? '2px' : `${adjustedSize}%`,
+        });
+    }, [adjustedSize, color, cssProperty]);
+
+    return (
+        <div
+            className={classNames('ca-Bar', { 'is-horizontal': isHorizontal })}
```

Comment:
this has a `ca` namespace, are we using that anywhere else? do we want to apply that for the rest of these components?

Reply:
We aren't and we know there's some inconsistencies in the css naming right now but I don't really want to change too much from the original migrated components in this PR

Reply:
fair enough. since it shouldn't be used outside of the webapp, it should be safe to leave and address the inconsistencies at a later time